Client/Clique/Common/Extensions/Extensions.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  Extensions.swift
+//  Clique
+//
+//  Created by Infinum on 27.11.2022..
+//
+
+import UIKit
+import SkeletonView
+
+extension UIImageView {
+    
+    func circleImage() {
+        layer.borderWidth = 1
+        layer.masksToBounds = false
+        layer.borderColor = UIColor.systemGray5.cgColor
+        layer.cornerRadius = self.frame.height / 2
+        clipsToBounds = true
+    }
+}
+
+extension UIView {
+    
+    func skeletonableView() {
+        layer.masksToBounds = false
+        self.isSkeletonable = true
+        self.showAnimatedSkeleton(usingColor: .clouds,
+                                          transition: .crossDissolve(0.5))
+    }
```

Comment:
This is wrong. Each extension should be in it's own file. And if a file is named `UIImageView+Extension.swift` you would now that skeletonableView for sure is not under that file but `UIView+Extension`.